### PR TITLE
All CSS Fixed ( for now )  and Copyright Year Updated

### LINKED ISSUE #79: Related to all CSS issues
- [ ] Responsive Watermark at Bottom Left
- [ ] Password TextBox

Please add more to the list, if you find any!

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
fixes #79

### CODE REVIEW

**Comment by Siddhant-K-code** [comment]
@jackyzha0 please review!

File: frontend/src/components/Watermark.js [hunk 1]
```diff
@@ -3,30 +3,40 @@ import React from "react";
...
+      position: relative;
+      color: ${(p) => p.theme.colors.text};
     }
-`
-export const Watermark = () => <Logo>
+  }
+
+  @media (max-width: 960px) {
+    position: relative;
+    bottom: 6.5em;
```
Comment (jackyzha0) [inline frontend/src/components/Watermark.js:29]: can you remove the bottom attr here? I think it looks better without it

File: frontend/src/components/Options.js [hunk 2]
```diff
@@ -1,39 +1,45 @@
...
+    position: auto;
+    display: inline-flex;
+  }
+  @media (max-width: 650px) {
+    position: relative;
+    float: none !important;
+    width: 100%;
+    display: inline-table;
+    text-align: center;
+  }
```
Comment (jackyzha0) [inline frontend/src/components/Options.js:25]: image

any way we can fix this weird spacing issue for the language dropdown?

**Review by jackyzha0 — COMMENTED** [review]
rest looks good!

File: frontend/src/components/Options.js (hunk 2 above)
Comment (Siddhant-K-code) [reply frontend/src/components/Options.js:25]: I also didn't get, what causes this? I will look into it.

Comment (Siddhant-K-code) [reply frontend/src/components/Options.js:25]: its not a CSS fault i guess.

Comment (jackyzha0) [reply frontend/src/components/Options.js:25]: ok I found a way to fix it, leaving a review with required changes

Comment (Siddhant-K-code) [reply frontend/src/components/Options.js:25]: can you please suggest it, so I can do it in the same only.

File: frontend/src/components/Options.js [hunk 3]
```diff
@@ -1,39 +1,45 @@
...
+  flex-direction: row;
+  transform: translateY(0.2em);
 
-    @media (max-width: 850px) {
-        float: none !important;
-    }
-`
+  & > *:not(:first-child) {
+    margin-left: 2em;
+  }
```
Comment (jackyzha0) [inline frontend/src/components/Options.js:13]:
```suggestion
  @media (min-width: 650px) {
    & > *:not(:first-child) {
      margin-left: 2em;
    }
  }
```

File: frontend/src/components/Options.js [hunk 4]
```diff
@@ -1,39 +1,45 @@
...
-                value={expiry}
-                onChange={onExpiryChange}
-                id="expiryInput" />
-        </Flex>
-    );
-}
+  @media (max-width: 850px) {
+    position: auto;
+    display: inline-flex;
+  }
```
Comment (jackyzha0) [inline frontend/src/components/Options.js:18]:
```suggestion
```

File: frontend/src/components/Options.js (hunk 2 above)
Comment (jackyzha0) [inline frontend/src/components/Options.js:25]:
```suggestion
  @media (max-width: 650px) {
    position: relative;
    float: none !important;
    flex-direction: column;
  }
```

**Comment by Siddhant-K-code** [comment]
@jackyzha0 Please Review. Looks fine now. 

image

image

**Review by jackyzha0 — APPROVED** [review]
clean stuff, thanks for helping with this! 🧼

**Pull request merged**